BrainCloudClient/tests/TestEvent.cs

```diff
@@ -89,6 +90,63 @@ public void TestDeleteIncomingEvent()
 
             tr.Run();
         }
+        
+        [Test]
+        public void TestDeleteIncomingEvents()
+        {
+            TestResult tr = new TestResult(_bc);
+            
+            string eventId1 = SendDefaultMessage();
+            string eventId2 = SendDefaultMessage();
+            string[] eventIds = {eventId1,eventId2};
+            
+            _bc.EventService.DeleteIncomingEvents(
+                eventIds,
+                tr.ApiSuccess, tr.ApiError);
+            tr.Run();
+            
+            _bc.EventService.GetEvents();
+            tr.RunExpectFail();
+        }
+        
+        [Test]
+        public void TestDeleteIncomingEventsOlderThan()
+        {
+            TestResult tr = new TestResult(_bc);
+
+            SendDefaultMessage();
+            
+            DateTime now = DateTime.UtcNow;
+            int dateInMillis = new DateTimeOffset(now).Millisecond;
+            
+            _bc.EventService.DeleteIncomingEventsOlderThan(
+                dateInMillis,
+                tr.ApiSuccess, tr.ApiError);
+            tr.Run();
+            
+            _bc.EventService.GetEvents();
+            tr.RunExpectFail();
+        }
+        
+        [Test]
+        public void TestDeleteIncomingEventsByTypeOlderThan()
+        {
+            TestResult tr = new TestResult(_bc);
+
+            string eventType = SendDefaultMessage();
```

Comment:
Wrong. SendDefaultMessage returnes an event id, not an event type. The event type is a member variable:
```
private readonly string _eventType = "test";
```